test/meta-colony.js

```diff
@@ -93,54 +108,57 @@ contract("Meta Colony", accounts => {
       expect(newSkill3.nParents).to.eq.BN(1);
       expect(newSkill3.nChildren).to.be.zero;
 
-      // Check rootSkill.nChildren is now 3
+      // Check rootSkill.nChildren is now 4
       const rootSkill = await colonyNetwork.getSkill(1);
-      expect(rootSkill.nChildren).to.eq.BN(3);
+      expect(rootSkill.nChildren).to.eq.BN(4);
 
       // Check rootSkill.children contains the ids of the new skills
-      const rootSkillChild1 = await colonyNetwork.getChildSkillId(1, 0);
+      const rootSkillChild1 = await colonyNetwork.getChildSkillId(1, 1);
       expect(rootSkillChild1).to.eq.BN(4);
-      const rootSkillChild2 = await colonyNetwork.getChildSkillId(1, 1);
+      const rootSkillChild2 = await colonyNetwork.getChildSkillId(1, 2);
       expect(rootSkillChild2).to.eq.BN(5);
-      const rootSkillChild3 = await colonyNetwork.getChildSkillId(1, 2);
+      const rootSkillChild3 = await colonyNetwork.getChildSkillId(1, 3);
       expect(rootSkillChild3).to.eq.BN(6);
     });
 
-    it("should NOT be able to add a child skill for a non existent parent", async () => {
-      // Add 2 skill nodes to root skill
-      await metaColony.addGlobalSkill(1);
-      await metaColony.addGlobalSkill(1);
+    it("should NOT be able to add a domain that has a non existent parent", async () => {
+      // Add 2 skill nodes to skill corresponding to root domain
+      await metaColony.addDomain(1);
+      await metaColony.addDomain(1);
 
-      await checkErrorRevert(metaColony.addGlobalSkill(6), "colony-invalid-skill-id");
+      await checkErrorRevert(metaColony.addDomain(6), "colony-domain-does-not-exist");
       const skillCount = await colonyNetwork.getSkillCount();
       expect(skillCount).to.eq.BN(5);
     });
 
-    it("should NOT be able to add a child skill to a local skill parent", async () => {
+    it.skip("should NOT be able to add a child skill to a local skill parent", async () => {
```

Comment:
why skipped 🤔

Reply:
🤔

Reply:
This function no longer takes a parent parameter, so there's no way to try and give it an ID corresponding to a local skill as a parent.

Reply:
Yeah, delete it then. I think the practice of skipping tests stayed with us since we were reluctant to delete reputation mining tests that took a while to write yet were obsoleted by added logic. We should clean that all up perhaps.

Reply:
After some thought, they're both reachable, so I've replaced the tests